sdk/appconfiguration/test-resources.json

```diff
@@ -125,10 +126,67 @@
             "apiVersion": "2016-10-01",
             "location": "[parameters('location')]",
             "dependsOn": [
-              "[resourceId('Microsoft.KeyVault/vaults', parameters('baseName'))]"
+                "[resourceId('Microsoft.KeyVault/vaults', parameters('baseName'))]"
             ],
             "properties": {
-              "value": "Very secret value"
+                "value": "Very secret value"
+            }
+        },
+        {
+            "type": "Microsoft.AppConfiguration/configurationStores/keyValues",
```

Comment:
Is it possible to set these keyvalues programmatically in test? We usually setup basic resources in this file, not preparing data for tests.

Reply:
I am currently doing it in both. But I think this is the correct way to do it. How else do you validate that it actually works? See https://github.com/Azure/azure-sdk-for-python/issues/31326, you have tests for feature flags, but they don't actually work. You are just validating you are consist within the SDK not with the service.